terraform/modules/filecoin_node/variable.tf

```diff
@@ -4,7 +4,7 @@ variable "image" {
 }
 
 variable "name" {
-  description = "The name of Forest Droplet"
+  description = "The name of Forest Droplet User"
```

Comment:
Name of the droplet user? Wasn't it right before?

Reply:
What's a droplet user? Is this variable also used for the username on the droplet?